structuralcodes/codes/ec2_2004/_concrete_creep_and_shrinkage.py

```diff
@@ -0,0 +1,534 @@
+"""Collection of functions from EUROCODE 1992-1-1:2004 related to creep and
+shrinkage.
+"""
+
+from __future__ import annotations  # To have clean hints of ArrayLike in docs
+
+import typing as t
+
+import numpy as np
+import numpy.typing as npt
+
+ALPHA_CEMENT_DICT = {'R': 1.0, 'N': 0.0, 'S': -1.0}
```

Comment:
Should we add a comment here indicating the reference to the code? Like defined in (B.9)?

Reply:
Good point. Fixed in a2dd665.